plugins/nanospell/plugin.js

```diff
@@ -91,7 +88,6 @@
 				node.getLength() > 0 &&  // and it's not empty
 				( !node.isReadOnly() ) &&   // or read only
 				isNotBookmark(node) && // and isn't a fake bookmarking node
-				!isInSpellCheckSpan && // it isn't in a spellcheck span
```

Comment:
removed, as skipping spellcheck spans can cause weird problems when previous typos are combined.

for example:

`qui^ckly` -> "ckly" gets picked up as a word.